doc/getting_started.md

```diff
@@ -221,30 +221,32 @@ everything is working correctly:
 ```console
 Creating user 'ot' with UID=1000, GID=1000.
 ot@ot-sca:/repo$ cd cw/cw305/
-ot@ot-sca:/repo/cw/cw305$ ./simple_capture_traces.py 
-Connecting and loading FPGA... Done!
+ot@ot-sca:/repo/cw/cw305$ ./capture.py --cfg-file capture_aes.yaml capture aes-random
+Connecting and loading FPGA... SKIPPED! (see: /home/abdullah/workspace/ot-sca/cw/cw305/util/device.py:initialize_fpga)
 Initializing PLL1
-Programming OpenTitan with "objs/aes_serial_fpga_nexysvideo.bin"...
-Transferring frame 0x00000000 @ 0x00000000.
-Transferring frame 0x00000001 @ 0x000003D8.
-Transferring frame 0x00000002 @ 0x000007B0.
-Transferring frame 0x00000003 @ 0x00000B88.
-Transferring frame 0x00000004 @ 0x00000F60.
-Transferring frame 0x00000005 @ 0x00001338.
-Transferring frame 0x00000006 @ 0x00001710.
-Transferring frame 0x00000007 @ 0x00001AE8.
-Transferring frame 0x00000008 @ 0x00001EC0.
-Transferring frame 0x00000009 @ 0x00002298.
-Transferring frame 0x0000000A @ 0x00002670.
-Transferring frame 0x8000000B @ 0x00002A48.
-Serial baud rate = 38400
-Serial baud rate = 115200
-Scope setup with sampling rate 100003051.0 S/s
-Target simpleserial version: z01 (attempts: 2).
-Using key: b'2b7e151628aed2a6abf7158809cf4f3c'
+WARNING:ChipWhisperer NAEUSB:Your firmware is outdated - latest is 1.2. Suggested to update firmware, as you may experience errors
+See https://chipwhisperer.readthedocs.io/en/latest/api.html#firmware-update
+WARNING:ChipWhisperer Scope:
+                ADC frequency exceeds specification (200 MHz). 
+                This may or may not work, depending on temperature, voltage, and luck.
+                It may not work reliably.
+                You can run scope.adc_test() to check whether ADC data is sampled properly by the FPGA,
+                but this doesn't fully verify that the ADC is working properly.
+                
+Programming OpenTitan with "objs/aes_serial_fpga_cw310.bin"...
+Transferring frame 0x00000000 @             0x00000000.
+Transferring frame 0x00000001 @             0x000007D8.
+Transferring frame 0x00000002 @             0x00000FB0.
+Transferring frame 0x00000003 @             0x00001788.
+Transferring frame 0x00000004 @             0x00001F60.
+Transferring frame 0x00000005 @             0x00002738.
+Transferring frame 0x80000006 @             0x00002F10.
+Scope setup with sampling rate 200009376.0 S/s
 Reading from FPGA using simpleserial protocol.
-Checking version: 
-Capturing: 100%|████████████████████████████| 5000/5000 [00:55<00:00, 90.34it/s]
+Target simpleserial version: z01 (attempts: 1).
+Using key: b'2b7e151628aed2a6abf7158809cf4f3c'                                  
+Capturing: 100%|██████████████████████████████| 100/100 [00:01<00:00, 51.46it/s]
+Created plot with 10 traces: /home/abdullah/workspace/ot-sca/cw/cw305/projects/sample_traces_aes.html
```

Comment:
maybe you can just replace `abdullah` here with `user` to make the guide more generic.